nslsii/iocs/tests/test_epstwostate_ioc.py

```diff
@@ -43,11 +43,17 @@ def test_epstwostate_ioc():
     stdout = subprocess.PIPE
     stdin = None
 
+    '''
     ioc_process = subprocess.Popen([sys.executable, '-m',
                                     'caproto.tests.example_runner',
                                     'nslsii.iocs.eps_two_state_ioc_sim'],
                                    stdout=stdout, stdin=stdin,
                                    env=os.environ)
+    '''
```

Comment:
As above please remove commented out sections and add a description of why this change is necessary

Reply:
See above comments associated with temperature_contollers_test